playbooks/cert_renewal.yml

```diff
@@ -0,0 +1,31 @@
+---
+# you must pass two variables:
+# pass the first element of the FQDN as '-e host_name=value'
+# for example: '-e host_name=fun' would generate a cert for fun.princeton.edu
+# also pass the path of the directory where you want the key and cert to live
+# with no trailing slash
+# for example: '-e cert_path=/home/foo/ssl_certs'
```

Comment:
@acozine it would be nice to have a full example and also a link for what to do with the CSR once you have it.

Reply:
@carolyncole thanks for the review - I've added a full example and some more documentation (plus the ability to add a SAN). Take a look and see what you think.